neqo-transport/src/cc/classic_cc.rs

```diff
@@ -275,21 +314,32 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
         pto: Duration,
         lost_packets: &[sent::Packet],
         now: Instant,
+        cc_stats: &mut CongestionControlStats,
     ) -> bool {
         if lost_packets.is_empty() {
             return false;
         }
 
-        for pkt in lost_packets.iter().filter(|pkt| pkt.cc_in_flight()) {
-            qdebug!(
-                "packet_lost this={self:p}, pn={}, ps={}",
-                pkt.pn(),
-                pkt.len()
-            );
-            // BIF is set to 0 on a path change, but in case that was because of a simple rebinding
-            // event, we may still declare packets lost that were sent before the rebinding.
-            self.bytes_in_flight = self.bytes_in_flight.saturating_sub(pkt.len());
+        self.cleanup_maybe_lost_packets(now, pto);
```

Comment:
Say we  get a million losses early on, but no additional one. Then `cleanup_maybe_lost_packets` would never be called again, and thus the packets would never be cleaned up. Is that a problem? Would it make more sense to do this in `on_packets_acked`?